app/CMakeLists.txt

```diff
@@ -464,15 +464,13 @@ if (IOS)
 
   # Generate Framewrok Headers
   foreach(hfile ${internal_HDRS})
-    message(STATUS "hfile: ${hfile}")
     string(FIND ${hfile} "firebase/" pos_firebase)
```

Comment:
As mentioned in the other PR, will be great to have a simple example in comments on various string transformations.